cerberus-api-tests/src/main/groovy/com/nike/cerberus/api/CerberusCompositeApiActions.groovy

```diff
@@ -33,10 +33,23 @@ class CerberusCompositeApiActions {
         // Read that the node was updated
         def resp2 = readSecretNode(path, cerberusAuthToken)
         assertEquals(value2, resp2?.'data'?.'value')
+
+        // Since the versions are sorted by timestamp, sleep is used to ensure correct order
+        sleep(1000)
```

Comment:
Did you try something shorter? I was thinking 100 would be more than enough.

This will increase the amount of time we have to wait when running the tests.

Reply:
Tried 500 and ran into the same issue